aries-site/src/pages/whats-new.mdx

```diff
@@ -53,6 +53,55 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## Febuary 22-26, 2021
+
+### New Spinner Component on Grommet.
+
+The new Spinner component has been released on grommet stable [branch](https://github.com/grommet/grommet/pull/4981).
+
+## Febuary 15-19, 2021
+
+### New FileInput Component
+- Check out the new [FileInput](https://v2.grommet.io/fileinput) Component on the grommet stable branch.
+
+### New Pagination Component 
+- The new [Pagination Component](https://v2.grommet.io/pagination) is ready to be used!
+
+### Re-align button sizes
+
+- Adjusted Button sizes to use [primary](https://github.com/grommet/hpe-design-system/pull/1433).
```

Comment:
I don't think we need to share this one, it's just regarding a tiny site example but not really a change someone would see in their app

Reply:
okay that's fine I wasn't sure where we draw the line

Reply:
No worries! I think in general we are thinking about What's New as a sort of "weekly release notes" in a sense, so any change that will affect how a team's app might look/new guidance on how things should behave